packages/idyll-components/src/scroller.js

```diff
@@ -92,6 +94,12 @@ class Scroller extends React.Component {
     }
   }
 
+  handleStepProgress(response) {
+    const progress = { response };
+    const update = { progress: progress * 100 };
```

Comment:
At this point the value of `progress` will be `{ response: valueOfTheResponse }`, and so multiplying by 100 will lead to `NaN`.

Currently we have

```js
const progress = { response }
```

which is shorthand for:

```js
const progress = { response: response }
```

but it should instead be
```js
const  { progress } = response

// this is equivalent to:
const progress = response.progress
```

Reply:
I didn't check it properly, my bad. Fixed it now!